mbed_build/_internal/mbed_tools/export.py

```diff
@@ -26,7 +26,9 @@
     help="The toolchain you are using to build your app.",
 )
 @click.option("-m", "--mbed_target", required=True, help="A build target for an Mbed-enabled device, eg. K64F")
-@click.option("-p", "--project-path", required=True, help="Path to local Mbed project")
+@click.option(
+    "-p", "--project-path", default=".", help="Path to local Mbed project. By default is the current working directory."
```

Comment:
Should we make this a `click.Path` to increase portability?

Reply:
bd45a03